app/views/layouts/godmin/_layout.html.erb

```diff
@@ -7,8 +7,9 @@
     <%= csrf_meta_tags %>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <%= yield :head %>
+    <img src="/logo.jpg" alt="logo" width="128" height="128"/>
```

Comment:
The initial image you have for the logo is not a perfect square so this would distort the image. I would just set the `height`.